docs/DeveloperGuide.md

```diff
@@ -302,32 +302,126 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+**Use case 1: View List of Customers**
 
-**Use case: Delete a person**
+* **System**: SellSavvy
+* **Actor**: USer
+* **Use Case**: UC01 - View List of Customers
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User chooses to view the list of customers.
+2.  SellSavvy retrieves all customers from the database.
+3.  SellSavvy displays all customers along with their details.
+4.  Use case ends.
 
-    Use case ends.
+**Use case 2: View List of Customers**
+
+* **System**: SellSavvy
+* **Actor**: USer
+* **Use Case**: UC02 - Add Customer
+
+**MSS**
+
+1.  User chooses to add a new customer.
+2.  SellSavvy validates the input and adds the customer to the database.
+3.  SellSavvy confirms the addition by displaying the newly added customer's details.
+4.  Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. SellSavvy detects required details missing.
+  * 1a1. SellSavvy displays an error message “Missing details of customer!”.
+  * 1a2. Use case ends.
 
-  Use case ends.
+**Use case 3: Delete Customer and All Orders Related to The Customer**
 
-* 3a. The given index is invalid.
+* **System**: SellSavvy
+* **Actor**: USer
+* **Use Case**: UC03 - Delete Customer and All Orders related to the customer
 
-    * 3a1. AddressBook shows an error message.
+**MSS**
 
-      Use case resumes at step 2.
+1.  User wants to delete a customer.
+2.  User chooses to view all customers.
+3.  SellSavvy displays the list of customers.
+4.  User finds the customer index from the list.
+5.  User deletes the customer by his index.
```

Comment:
`his` may be slightly unclear. Perhaps specifying as `customer index` instead may be clearer?
```suggestion
5.  User deletes the customer by the customer's index in the list.
```

List here would follow from UC01's output list